third_party/angle/tnt/CMakeLists.txt

```diff
@@ -0,0 +1,5 @@
+cmake_minimum_required(VERSION 3.19)
+project(angle)
+
+add_library(angle INTERFACE)
+target_include_directories(angle INTERFACE ${ANGLE_INCLUDE_DIR})
```

Comment:
I would leave some commented code here, which helps anyone wanting to try out sample targets using ANGLE, (i.e. linking dependecies). For example:
```cmake
# Uncomment the following lines and specify your folder containing ANGLE libraries to enable linking sample projects using ANGLE:
# set(ANGLE_LIBS_DIR "")
# target_link_directories(angle INTERFACE ${ANGLE_LIBS_DIR})
# target_link_libraries(angle INTERFACE "libEGL.dll.lib" "libGLESv2.dll.lib")
```